src/dataflow/src/source/postgres.rs

```diff
@@ -109,6 +101,35 @@ impl PostgresSourceReader {
             ))
             .await;
 
+        // Get all the relevant tables for this publication
+        let publication_tables = client
+            .simple_query(&format!(
+                "SELECT
```

Comment:
Even though we can't reuse the call to Postgres to get the publication tables, do you think it might be worthwhile to extract the query strings themselves into variables? I guess they won't change much, but just to ensure that `postgres-util` and `source/postgres` don't fall out of sync. (Totally optional though, just a thought!)

Reply:
that's a good idea. I'll try to dedup them